astropy/io/votable/tests/vo_test.py

```diff
@@ -151,19 +151,16 @@ def _test_regression(tmpdir, _python_based=False, binary_mode=1):
         get_pkg_data_filename(
             'data/regression.bin.tabledata.truth.{0}.xml'.format(
                 votable.version)),
-        'rt', encoding='utf-8') as fd:
+            'rt', encoding='utf-8') as fd:
         truth = fd.readlines()
     with io.open(str(tmpdir.join("regression.bin.tabledata.xml")),
                  'rt', encoding='utf-8') as fd:
         output = fd.readlines()
 
     # If the lines happen to be different, print a diff
     # This is convenient for debugging
-    for line in difflib.unified_diff(truth, output):
-        sys.stdout.write(
-            line.
-            encode('unicode_escape').
```

Comment:
Note: There must had been reason to do the "encode" and "replace" but let's see what happens without those.